test/test_utils.hpp

```diff
@@ -162,7 +162,7 @@ struct test_range_algo_1
         using S = meta::_t<sentinel_type<I>>;
         using R = decltype(algo_(begin, end, rest...));
         Algo algo = algo_;
-        auto check_algo = [=](function_ref<void(R)> const & check)
+        auto check_algo = [algo, begin, end, rest...](function_ref<void(R)> const & check)
```

Comment:
Should we init-capture `algo = algo_` here and remove line 164? We don't need to support C++11 on v1.0-beta.